components/tidb_query_datatype/src/codec/mysql/enums.rs

```diff
@@ -91,6 +84,19 @@ impl<'a> EnumRef<'a> {
     }
 }
 
+impl<'a> ToString for EnumRef<'a> {
+    fn to_string(&self) -> String {
+        if self.value == 0 {
+            return String::new();
+        }
+
+        let buf = &self.data[self.value - 1];
```

Comment:
Is the `value` always valid? Or allow panic here?

Reply:
Yes, value always valid